cas-server-core-tickets/src/main/java/org/jasig/cas/ticket/registry/AbstractTicketRegistry.java

```diff
@@ -38,6 +42,26 @@
     @Qualifier("ticketCipherExecutor")
     private CipherExecutor<byte[], byte[]> cipherExecutor;
 
+    @Nullable
+    @Autowired(required = false)
+    @Qualifier("ticketDelegators")
+    private List<Pair<Class<? extends Ticket>, Constructor<? extends AbstractTicketDelegator>>> ticketDelegators = new ArrayList<>();
```

Comment:
Yes, autowiring won't work, `@Resource` is necessary, though it becomes mandatory which is not our use case. I will remove autowiring for now.